<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TT_DialogV0</class>
 <widget class="QDialog" name="TT_DialogV0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculate V0</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbStation">
       <property name="text">
        <string>Station :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbStation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbReferences">
     <property name="text">
      <string>References to use :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>230</width>
        <height>93</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbCalculate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Calculate V0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbV0">
       <property name="text">
        <string>V0 :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leV0">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TT_DialogV0</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TT_DialogV0</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
